src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -449,7 +449,8 @@ public static IEnumerable<string> GetAllProjectFileNamesWithXBuild(string soluti
             string solutionFile,
             string msbuildPath)
```

Comment:
Can you rename the parameter here? 
It'll improve clarity :)

Reply:
renamed